spring-xd-deployment.tex

```diff
@@ -1,30 +1,18 @@
 \section{Deployment}
 This section describes deployment architectures that are possible with Spring XD.
-Note that this is not an exhaustive list of all deployment types. See
-section~\ref{sec:Use Cases} for descriptions of real world deployments.
+While XD does not subscribe to a specific deployment model (Lambda, 
+Reactive or other), it does allow for a user to architect the deployment type 
+that meets their needs.  Note that this is not an exhaustive list of all 
+deployment types, see section~\ref{sec:Use Cases} for descriptions of real 
+world deployments.
 
 \subsection{Lambda Architecture}
 
-The Lambda Architecture, introduced by Nathan Marz \cite{lambda-architecture-paper}
```

Comment:
I would remove this section header altogether and simply have three dedicated subsections for streaming, batch and stream+batch.